modules/pam_faillock/pam_faillock.c

```diff
@@ -374,7 +374,9 @@ write_tally(pam_handle_t *pamh, struct options *opts, struct tally_data *tallies
 		}
 		close(audit_fd);
 #endif
-		if (!(opts->flags & FAILLOCK_FLAG_NO_LOG_INFO)) {
+		if ((!(opts->flags & FAILLOCK_FLAG_NO_LOG_INFO)) && ((opts->uid != 0)  ||
+		    (opts->flags & FAILLOCK_FLAG_DENY_ROOT))) {
```

Comment:
Consider the following formatting:
```
if (!(opts->flags & FAILLOCK_FLAG_NO_LOG_INFO) &&
    ((opts->uid != 0) || (opts->flags & FAILLOCK_FLAG_DENY_ROOT))) {
```

Reply:
Hi Dmitry,

Thank you for your feedback.

I have made the necessary changes.

Can you kindly review again.

Thank you!